src/analyze/State.re

```diff
@@ -226,12 +226,21 @@ let newBsPackage = (~reportDiagnostics, state, rootPath) => {
         };
         spec |?> Json.get("module") |?> Json.string
       } |? "commonjs";
-      let flags = jsPackageMode == "es6" ? [
-        "-bs-package-name",
-        config |> Json.get("name") |?> Json.string |? "unnamed",
-        "-bs-package-output",
-        "es6:node_modules/.lsp",
-        ...flags] : flags;
+      let flags = switch (jsPackageMode) {
```

Comment:
@bloodyowl there seems to be some repetition here. What do you think about changing to something like:

```js
[
  "-bs-package-name",
  config |> Json.get("name") |?> Json.string |? "unnamed",
  ...if (jsPackageMode == "es6") {
       ["-bs-package-output", "es6:node_modules/.lsp", ...flags];
     } else {
       flags;
     },
];
```